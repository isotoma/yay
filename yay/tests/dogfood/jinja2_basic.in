wibble: world

hello.jinja: |
    hello {{ wibble }}

wibble: lol

finally: it said ${hello}

