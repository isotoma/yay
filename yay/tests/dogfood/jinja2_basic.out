finally: it said hello lol
hello: hello lol
wibble: lol

